cmd/dependabot/internal/cmd/update.go

```diff
@@ -38,6 +38,15 @@ type UpdateFlags struct {
 	apiUrl          string
 }
 
+// A map of package manager names to credential type
+var packageManagerCredentialType = map[string]string{
```

Comment:
I was confused why all of the types weren't listed. Can this be renamed and/or commented to specify it's Azure supported credential types?